README.md

```diff
@@ -82,7 +83,8 @@ volumeBindingMode: WaitForFirstConsumer
 parameters:
   storagePool: local
 ```
-Notice the storagePool parameter. This lets the provisioner know which pool to use. Once multiple pools are supported you can create multiple storage classes, each pointing to a different pool.
+Notice the storagePool parameter. This lets the provisioner know which pool to use. You can define multiple storage pools each
```

Comment:
multiple pools on a single storageClass?

Reply:
No each pool has its own storage class on the hpp side. Notice the storagePool parameter in the storage class definition.